src/fuse_api.pxi

```diff
@@ -467,8 +467,11 @@ cdef session_loop_mt(workers):
         stdlib.free(wd)
 
 
-def stop_fuse_from_another_thread():
-    '''Stop the FUSE main loop from another thread.
+def stop():
+    '''Stop the FUSE main loop.
+
+    This function is thread safe. Note once the `main` function has returned
```

Comment:
I think you forgot a "that" ("Note that ..."). Or just omit the preable and say "Once the main function.."

Reply:
done